pages/clubs-guides/clubs-guides.jsx

```diff
@@ -0,0 +1,99 @@
+var React = require('react');
+var HeroUnit = require('../../components/hero-unit.jsx');
+
+var GuideList = React.createClass({
+  getInitialState: function(){
+    return {
+      guides : []
+    }
+  },
+  setData : function(data){
+    this.setState({guides : data });
+  },
+  fetchJSON : function(path,callback){
+    var httpRequest = new XMLHttpRequest();
+    httpRequest.onreadystatechange = function() {
+      if(httpRequest.readyState === 4) {
+        if(httpRequest.status === 200) {
+          var data = JSON.parse(httpRequest.responseText);
+          if (callback) callback(data);
+        }
+      }
+    };
+    httpRequest.open('GET', path);
+    httpRequest.send();
+  },
+  componentDidMount: function() {
+    this.fetchJSON("https://mozilla.github.io/learning-networks/clubs/clubs-resources.json",this.setData);
+  },
+  render: function(){
```

Comment:
nit: Just a suggestion, I think it would look a bit better if you make the `render` function a little more compact? Move the different `array.map` pieces of logic into different functions in this object and let your render just be something like:
```js
return (
  <div className="guideList">
    { this.getCategories() }
  </div>
);
```